### ISSUE TITLE
Add accessible descriptions to this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#180]
We are ready to add accessible descriptions to this sim.  Here is a link to the description spreadsheet that defines various descriptions for the sim.

https://docs.google.com/spreadsheets/d/1AwlqM8DUchUunzUHyK63xXIIjVtgkV6Q55oDLlKC9hg/edit#gid=0

@mbarlow12 said he could work on this, thanks!

**Comment by mbarlow12** [comment]
Completed the help text in the above commits. We can review at

https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.5.0-dev.4/phet/molecules-and-light_a11y_view.html

Up next is handling the alerts.

**Comment by mbarlow12** [comment]
I noticed this text in the description [spreadsheet](https://docs.google.com/spreadsheets/d/1AwlqM8DUchUunzUHyK63xXIIjVtgkV6Q55oDLlKC9hg/edit#gid=0),

`* In observation window, {{ultraviolet}} light source points direclty at {{an}} {{ozone}} molecule and photons are being emitted at {{slow}} rate towards the molecule.`

This appears to be a 3rd case for the dynamic text for the Observation Window, so I'm curious if it should be implemented, or if it's a note for further investigation? @terracoda?

**Comment by terracoda** [comment]
@mbarlow12 are you referring to cell C9 under Linear Description tab? Where are the other 2 descriptions for the Observation Window?

Phrases in C9 are not alerts by my examination. I believe that the 2 cases there are dynamic descriptions and intended to describe the state of the observation window. If it the light source is off, you get various possibilities using the phrase for Case 1, and if the light source is on, you get various possibilities of the phrase labeled Case 2. You wouldn't get both at the same time.

You probably know that there are three tabs in the spreadsheet, one is titled Alert Description, so I think the alerts are in that tab.

Does that help?

**Comment by terracoda** [comment]
@mbarlow12, the link in https://github.com/phetsims/molecules-and-light/issues/180#issuecomment-375756966 is not working. Could post a new one when you have time?

**Comment by mbarlow12** [comment]
@terracoda It's actually the cell G9 in the 'Notes' column. There's an added placeholder for the rate of photon emission (e.g. {{slow}}) that does not appear in the C9 strings.

C9 Case 2:
```
In observation window, {{ultraviolet}} light source emits photons directly at {{an}} {{ozone}} molecule.
```

G9
```
* In observation window, {{ultraviolet}} light source points direclty at {{an}} {{ozone}} molecule and photons are being emitted at {{slow}} rate towards the molecule.
```

It's unclear if this text is supposed to appear in the html or if it's just a note from the design process/dicsussion.

**Comment by mbarlow12** [comment]
Here's the latest dev release:
https://bayes.colorado.edu/dev/html/molecules-and-light/1.5.0-dev.5/phet/molecules-and-light_a11y_view.html

**Comment by terracoda** [comment]
@mbarlow12, oh, I see, great question! I am pretty sure the notes column, is just "notes" that you can ignore. I think in this case, the note is a previous iteration.

I think you just need either case 1 or case 2 that are actually in cell C9.

**Comment by terracoda** [comment]
@mbarlow12, I had a quick peak at the a11y view. The content is looking great! Nice work so far!

**Comment by mbarlow12** [comment]
Thanks for the clarification! Alerts are almost done—they just need some clean up.

**Comment by terracoda** [comment]
@mbarlow12, re https://github.com/phetsims/molecules-and-light/issues/180#issuecomment-382898599, I made a quick stab at Methane, based on the other descriptions.

**Comment by ariel-phet** [comment]
Marking deferred until someone is assigned to do this a11y work

**Comment by terracoda** [comment]
I'm working on a github issue template for tracking A11y design and implementation.
We should use it, but it is not quite ready. Please see:
https://github.com/phetsims/a11y-research/blob/master/issue-templates/description-design-template.md

**Comment by terracoda** [comment]
@Matthew-phet and @zepumph, the issue template may be good enough to try to use with Moleclues and Light. 

One possible way to move forward, is to you close this issue, and reference it in a new issue that is populated with the checklist.

**Comment by terracoda** [comment]
@Matthew-phet and @zepumph, I think the issue template is good enough to use, and many things can already be checked off.
https://github.com/phetsims/a11y-research/blob/master/issue-templates/description-design-template.md

**Comment by zepumph** [comment]
I agree we should close this issue and create a new one to track description progress with the template.

**Comment by terracoda** [comment]
@Matthew-phet, I'll close this issue and we can continue tracking progress in https://github.com/phetsims/molecules-and-light/issues/201
